### PR TITLE
IRIS-4081 Move On Site Notifications CSS to Design System

### PR DESCRIPTION
**Pull request by slayful**

### CODE REVIEW

File: app/styles/base-styles/_z-idex-scale.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+$z-1   : 100;
+$z-2   : 200;
+$z-3   : 300;
+$z-4   : 400;
+$z-5   : 500;
+$z-6   : 600;
+$z-7   : 700;
+$z-8   : 800;
+$z-9   : 900;
+$z-max : 10000;
```
Comment (kvas-damian) [inline app/styles/base-styles/_z-idex-scale.scss]: `10000` or `1000`?

File: app/styles/base-styles/_z-idex-scale.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Z-Index Scale
+// - - - - - - - - - - - - - - - - - - - - - - - - -
+
+// Never --ever-- implement z-index that doesn't use this scale, if you have a use-case that
+// for some reason falls out of this scale, ask for guidance before implementing one-offs.
```
Comment (kvas-damian) [inline app/styles/base-styles/_z-idex-scale.scss]: who should I ask then? ;)

File: app/styles/base-styles/_z-idex-scale.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// Z-Index Scale
```
Comment (kvas-damian) [inline app/styles/base-styles/_z-idex-scale.scss]: It'd be great to have a page on wikiadesignsystem.com which has some short documentation

File: app/styles/components/_avatar-stack.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+	&__avatar {
+		color: $wds-color-light-c4;
+	}
+
+	@for $i from 1 to 6 {
+		.wds-avatar-stack__avatar:nth-child(#{$i}) {
+			position: relative;
+			transform: translate(($i - 1) * - 15px);
+			z-index: 5 - $i;
```
Comment (kvas-damian) [inline app/styles/components/_avatar-stack.scss]: isn't it against comment in _z-idex-scale.scss ?

File: app/styles/components/_menu-chevron.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.wds-menu-chevron {
+	float: right;
+	margin-top: 5px;
+	transform: rotate(-90deg);
```
Comment (kvas-damian) [inline app/styles/components/_menu-chevron.scss]: it's not obvious how to use this component. Probably we need a page with some examples

File: app/styles/components/_notifications.scss [hunk 6]
```diff
@@ -0,0 +1,153 @@
+$notifications-dropdown-content-color: $wds-color-light-c7;
+$wds-icon-color: $wds-color-light-c5;
+
+$content-padding: 18px;
+
+.wds-notifications {
```
Comment (kvas-damian) [inline app/styles/components/_notifications.scss]: should this be component dedicated to notifications only? Maybe we should extract a part that can be used in other use-cases?

File: app/styles/components/_notifications.scss [hunk 7]
```diff
@@ -0,0 +1,153 @@
+$notifications-dropdown-content-color: $wds-color-light-c7;
```
Comment (kvas-damian) [inline app/styles/components/_notifications.scss]: can we move those variables into styles block, they are not exposed publicly then.

File: app/styles/components/_spinner.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+	@keyframes dash {
+		to {
+			stroke-dashoffset: 0;
+		}
+	}
+
+	animation: rotator $duration linear infinite;
+	left: 50%;
+	margin: -33px 0 0 -33px;
```
Comment (kvas-damian) [inline app/styles/components/_spinner.scss]: I don't see size defined for this spinner. Are hardcoded values ok for every size? Maybe we should use `translate` to move it half left and top? Or maybe you'd use `flexbox`?

File: app/styles/components/_spinner.scss [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+	}
+
+	&__stroke {
+		-webkit-backface-visibility: hidden;
+		animation: dash ($duration / 2.5) linear infinite alternate-reverse;
+		stroke: $spinner-color;
+	}
+
+	&__overlay {
+		background: rgba(white, .5);
```
Comment (kvas-damian) [inline app/styles/components/_spinner.scss]: can we use defined DS colors?

File: app/styles/components/_spinner.scss (hunk 8 above)
Comment (slayful) [inline app/styles/components/_spinner.scss]: The size is set by JS code (a couple of things have to be calculated for it to work).

This is pretty old CSS, I've given it a quick try and couldn't fix it without the margin. We've got one very big spinner and a couple of smaller ones and they seem to be working with this margin.

I'll note it and if I don't get back to it I'll create a ticket.

File: app/styles/base-styles/_z-idex-scale.scss (hunk 3 above)
Comment (slayful) [inline app/styles/base-styles/_z-idex-scale.scss]: https://wikia-inc.atlassian.net/browse/IRIS-4213

File: app/styles/components/_menu-chevron.scss (hunk 5 above)
Comment (slayful) [inline app/styles/components/_menu-chevron.scss]: https://wikia-inc.atlassian.net/browse/IRIS-4214

File: app/styles/components/_avatar-stack.scss (hunk 4 above)
Comment (slayful) [inline app/styles/components/_avatar-stack.scss]: Are you the right person to ask if this is ok?

Comment (kvas-damian) [inline app/styles/components/_avatar-stack.scss]: I'm not sure.
I hope it's ok for now.

**Pull request merged**